osu.Game/Localisation/LayoutSettingsStrings.cs

```diff
@@ -20,9 +20,9 @@ public static class LayoutSettingsStrings
         public static LocalisableString OsuIsRunningExclusiveFullscreen => new TranslatableString(getKey(@"osu_is_running_exclusive_fullscreen"), @"osu! is running in exclusive fullscreen, guaranteeing low latency!");
 
         /// <summary>
-        /// "Unable to run exclusive fullscreen. You&#39;ll still experience some input latency."
+        /// "Unable to run in exclusive fullscreen. You&#39;ll experience some input latency."
         /// </summary>
-        public static LocalisableString UnableToRunExclusiveFullscreen => new TranslatableString(getKey(@"unable_to_run_exclusive_fullscreen"), @"Unable to run exclusive fullscreen. You'll still experience some input latency.");
+        public static LocalisableString UnableToRunExclusiveFullscreen => new TranslatableString(getKey(@"unable_to_run_exclusive_fullscreen"), @"Unable to run in exclusive fullscreen. You'll experience some input latency.");
```

Comment:
How about "You may experience some input latency"?

Reply:
Oh yeah, that makes sense